neo/io/nwbio.py

```diff
@@ -434,10 +435,15 @@ def write_all_blocks(self, blocks, **kwargs):
         if "session_description" not in annotations:
             annotations["session_description"] = blocks[0].description or self.filename
             # todo: concatenate descriptions of multiple blocks if different
+        
         if "session_start_time" not in annotations:
-            raise Exception("Writing to NWB requires an annotation 'session_start_time'")
+            annotations["session_start_time"] = blocks[0].rec_datetime
+            if annotations["session_start_time"] is None:
+                raise Exception("Give a session start time")
```

Comment:
please use the same exception text as previously, i.e. "Writing to NWB requires an annotation 'session_start_time'"